code/modules/mob/living/simple_animal/hostile/syndicate.dm

```diff
@@ -116,7 +116,9 @@
 
 /mob/living/simple_animal/hostile/syndicate/melee/autogib/depot/Aggro()
 	. = ..()
-	if(target && istype(depotarea))
+	if(!istype(depotarea))
+		return .

```

Comment:
Explicit return not necessary as 
``` . = ..() ``` Is already done which already return the super call

Reply:
Changed from "return ." to "return".